mithril-test-lab/mithril-devnet/devnet-mkfiles.sh

```diff
@@ -329,6 +329,7 @@ echo "====================================================================="
 mkdir shelley
 curl -s ${ALONZO_GENESIS_URL} -o shelley/genesis.alonzo.spec.json
 curl -s ${CONWAY_GENESIS_URL} -o shelley/genesis.conway.spec.json
+mv shelley/genesis.conway.spec.json shelley/genesis.conway.spec.json.tmp && cat shelley/genesis.conway.spec.json.tmp | jq '. += {"genDelegs":{}}' > shelley/genesis.conway.spec.json && rm shelley/genesis.conway.spec.json.tmp
```

Comment:
Can you put a comment that explain why we have to do that ?

Reply:
Done :heavy_check_mark: